src/main.js

```diff
@@ -4,89 +4,94 @@
+    };
+    return bookObj;
   }
-}
 
-// ***************
-// main functions
-// ***************
+  saveLocal = () => {
+    const catchCollection = JSON.stringify(this.bookCollection);
+    localStorage.setItem('bookCollection', catchCollection);
+  }
 
-function bookTemplate(el) {
-  return `
-<h3 class="bookTitle">${el.title}</h3>
-<p class="bookAuthor">${el.author}</p>
-<button type='button' class="removeBookBtn">Remove</button>
-`;
-}
+  getLocal = () => {
+    if (localStorage.getItem('bookCollection')) {
+      this.bookCollection = JSON.parse(localStorage.getItem('bookCollection'));
+    }
+  }
 
-function renderBooks() {
-  $bookContainer.innerHTML = '';
-  bookCollection.forEach((el) => {
-    const article = document.createElement('article');
-    article.className = 'article-book';
-    article.innerHTML = bookTemplate(el);
-    $bookContainer.appendChild(article);
-  });
-}
+  // ***************
+  // main functions
+  // ***************
 
-function createRemoveBtn() {
-  const $removeBookBtn = document.querySelectorAll('.removeBookBtn');
-  $removeBookBtn.forEach((el, index) => {
-    el.addEventListener('click', () => {
-      bookCollection.splice(index, 1);
-      renderBooks();
-      createRemoveBtn();
-      saveLocal();
+  renderBooks = () => {
+    $bookContainer.innerHTML = '';
+    this.bookCollection.forEach((el) => {
+      const article = document.createElement('article');
+      article.className = 'article-book';
+      article.innerHTML = `
+  <h3 class="bookTitle">"${el.title}" by ${el.author}</h3>
+  <button type='button' class="removeBookBtn">Remove</button>
+  `;
+      $bookContainer.appendChild(article);
     });
-  });
-}
+  }
 
-function displayBookCollection() {
-  $bookContainer.innerHTML = '';
-  renderBooks();
-  createRemoveBtn();
-}
+  createRemoveBtn = () => {
+    const $removeBookBtn = document.querySelectorAll('.removeBookBtn');
+    $removeBookBtn.forEach((el, index) => {
+      el.addEventListener('click', () => {
+        this.bookCollection.splice(index, 1);
+        this.renderBooks();
+        this.createRemoveBtn();
+        this.saveLocal();
+      });
+    });
+  }
 
-function clearFields() {
-  $titleInput.value = '';
-  $authorInput.value = '';
-}
+  displayBookCollection = () => {
+    this.renderBooks();
+    this.createRemoveBtn();
+  }
 
-function addBook() {
-  if ($titleInput.value && $authorInput.value) {
-    bookCollection.push({
-      title: $titleInput.value,
-      author: $authorInput.value,
-    });
-    displayBookCollection();
-    saveLocal();
-    clearFields();
+  clearFields = () => {
+    $titleInput.value = '';
+    $authorInput.value = '';
   }
-}
 
-function main() {
-  getLocal();
-  displayBookCollection();
-  $addBookBtn.addEventListener('click', addBook);
+  addBook = () => {
+    if ($titleInput.value && $authorInput.value) {
+      const newBook = this.book($titleInput.value, $authorInput.value);
+      this.bookCollection.push(newBook);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `concat` over `push`, which doesn't mutate the array.